config/schedule.rb

```diff
@@ -15,4 +15,7 @@
 
   # make sure the daemon is running
   rake "ts:start"
-end
\ No newline at end of file
+
+  # Clean up expired media
+  runner "MediaCleanupJob.perform_later"
```

Comment:
I'm worried that this may end up spinning up two Rails processes at once. Could we perhaps create an 'hourly' rake task that does these things? Could perhaps use `ThinkingSphinx::RakeInterface#start` to do the Sphinx thing...